ksql-parser/src/main/java/io/confluent/ksql/parser/tree/AbstractStreamDropStatement.java

```diff
@@ -14,14 +14,65 @@
 
 package io.confluent.ksql.parser.tree;
 
+import com.google.common.base.MoreObjects;
+import java.util.Objects;
 import java.util.Optional;
 
 public abstract class AbstractStreamDropStatement extends Statement {
-  public AbstractStreamDropStatement(final Optional<NodeLocation> location) {
+
+  private final QualifiedName name;
+  private final boolean ifExists;
+  private final boolean deleteTopic;
+
+  public AbstractStreamDropStatement(final Optional<NodeLocation> location,
```

Comment:
Total nit but I think the preferred style is

```suggestion
public AbstractStreamDropStatement(
    final Optional<NodeLocation> location,
    final QualifiedName name,
    final boolean deleteTopic,
    final boolean ifExists
    ) {
```